tests/unit/customizations/test_waiters.py

```diff
@@ -10,13 +10,13 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
-import mock
-
 from botocore.waiter import WaiterModel
 from botocore.exceptions import DataNotFoundError
 
-from awscli.testutils import unittest, BaseAWSHelpOutputTest, \
-    BaseAWSCommandParamsTest
+from awscli.testutils import (
+        mock, unittest, 
```

Comment:
nit: We should get this line aligned with the line below. It also looks like there is extra white space at the EOL.